python/cudf/cudf/__init__.py

```diff
@@ -56,6 +56,8 @@
     StructDtype,
 )
 from cudf.core.groupby import Grouper
+
+# TODO: Math operations like add, arccos, etc. are not exposed in pandas' root namespace.
```

Comment:
I agree that we should plan to remove these at some point. However, it's not quite so easy as a simple removal because these are currently used (as part of a needlessly complex scheme) for implementing `__array_function__` and `__array_ufunc__`. Disentangling that is something that I started looking at, but since I was pretty tied up @shwina offered to take that project on since he was interested. I assume that he's been swamped and hasn't gotten around to it, but the upshot is that we can't remove these until that problem has been addressed.

If, for the sake of doctests, we would just want to remove the functions from the parent namespace without removing the implementations in core/ops.py, I would support changing the `__array_function__` and `__array_ufunc__` dispatches to perform the lookup in `cudf.core` instead of in `cudf` instead (or more likely in addition, unfortunately I don't think _all_ of the necessary functionality comes from core/ops.py but only a subset). That would be reasonable within the scope of this PR.

Reply:
I posted a comment here: https://github.com/rapidsai/cudf/issues/9083#issuecomment-1005175782

I'll remove the TODO and save this for future work. For now, I'll skip adding those functions to `__all__`.